src/MoBi.UI/Views/BaseModuleContentView.cs

```diff
@@ -10,11 +8,12 @@
 
 namespace MoBi.UI.Views
 {
-   public partial class CreateModuleView : BaseModalView, ICreateModuleView
+   public abstract partial class BaseModuleContentView<TDTO> : BaseModalView
```

Comment:
This view is pretty much identical for various presenters that enable users to select certain building blocks of a module

Create, Add, Clone as of now. I refactored it with a common base and minimal concrete implementation details